dashboard/config/scripts/flappy.script

```diff
@@ -1,5 +1,5 @@
 id '6'
-title 'Flappy Code'
+title 'Flappy Bird'
```

Comment:
Interesting.  I know in the past I've been explicitly told not to use the name Flappy Bird for this tutorial.